lib/src/Uncertainty/Model/StandardEvent.cxx

```diff
@@ -37,7 +41,7 @@ CLASSNAMEINIT(StandardEvent)
 
 /* Default constructor */
 StandardEvent::StandardEvent()
-  : Event()
+  : RandomVector(new EventRandomVector(CompositeRandomVector(IdentityFunction(1), RandomVector(Uniform())), Less(), 0.0))
```

Comment:
It should be Normal() instead of Uniform() as a StandardEvent is mostly expected to land into a Gaussian space.